### PR TITLE
Drag and drop handler fix

### PR DESCRIPTION
**Pull request by jacob-v-dam**
## Type
- Non critical bugfix

## Pull request description
New drag and drop handlers fixed

### CODE REVIEW

File: src/Backend/Core/Js/backend.js [hunk 1]
```diff
@@ -1883,7 +1883,7 @@ jsBackend.tableSequenceByDragAndDrop = {
           placeholder: 'dragAndDropPlaceholder',
           forcePlaceholderSize: true,
           stop: function (e, ui) {
-            jsBackend.tableSequenceByDragAndDrop.saveNewSequence($(this))
+            jsBackend.tableSequenceByDragAndDrop.saveNewSequence($(this).closest('table'))
```
Comment (carakas) [inline src/Backend/Core/Js/backend.js]: must be table.jsDataGrid

File: src/Frontend/Core/Header/Header.php [hunk 2]
```diff
@@ -119,7 +119,12 @@ public function __construct(KernelInterface $kernel)
         $this->meta = new MetaCollection();
 
         // add some default CSS files
-        $this->addCSS('/src/Frontend/Core/Layout/Css/screen.css');
+        $themeCss = $container->getParameter('site.path_www') . '/src/Frontend/Themes/' . Theme::getTheme() . '/Core/Layout/Css/screen.css';
```
Comment (carakas) [inline src/Frontend/Core/Header/Header.php]: why is this commit in here?
If I remember it correctly this has been fixed in fork in a different way so you should drop this commit

File: src/Backend/Modules/MediaLibrary/Domain/MediaGroup/MediaGroup.php [hunk 3]
```diff
@@ -163,6 +163,7 @@ function (MediaGroupMediaItem $connectedItem): MediaItem {
     public function getFirstConnectedMediaItem(): ?MediaItem
     {
         $connectedMediaItems = $this->getConnectedMediaItems();
+        $firstMediaItem = null;
```
Comment (carakas) [inline src/Backend/Modules/MediaLibrary/Domain/MediaGroup/MediaGroup.php]: this commit shouldn't be in here

Comment (jacob-v-dam) [reply src/Backend/Modules/MediaLibrary/Domain/MediaGroup/MediaGroup.php]: In deed, i noticed this i'm removing this pull request and use the right commits :)

File: src/Backend/Core/Js/backend.js (hunk 1 above)
Comment (jacob-v-dam) [reply src/Backend/Core/Js/backend.js]: Fixed

**Pull request merged**